packages/user/src/services/user-module.ts

```diff
@@ -96,6 +101,20 @@ export default class UserModuleService<
       populate: true,
     })
 
+    sharedContext.messageAggregator?.save(
```

Comment:
question: I think we should look into exposing a simpler API for the `messageAggregator`. If we will be using `buildEventMessages` everywhere, we might as well think in ways of baking it in.

Ideally, we can do something like:
```ts
 sharedContext.messageAggregator?.save(
        users.map((user) => ({
          eventName: UserEvents.CREATED,
          metadata: {
            service: this.constructor.name,
            action: "create",
            object: "user",
          },
          data: user.id,
        })
      )
    )
```

And also allow for a single event to be passed, as we currently do with our event bus:
```ts
sharedContext.messageAggregator?.save({
  eventName: UserEvents.CREATED,
  metadata: {
    service: this.constructor.name,
    action: "create",
    object: "user",
  },
  data: user.id,
})
```

cc @carlos-r-l-rodrigues